server/website/website/tasks/async_tasks.py

```diff
@@ -450,7 +459,7 @@ def map_workload(target_data):
             # the knob configurations attempted so far by the target.
             y_col = y_col.reshape(-1, 1)
             model = GPRNP()
```

Comment:
Could you please pass the parameters to GPRNP here ? e.g. length_scale=DEFAULT_LENGTH_SCALE, magnitude=DEFAULT_MAGNITUDE..   MAX_TRAIN_SIZE , BATCH_SIZE